command/login.go

```diff
@@ -69,17 +71,34 @@ var Login = cli.Command{
 			password = c.String("password")
 		}
 
-		token, err := m.Login(username, password)
+		token, err := m.Login(username, password, "")
+
+		// MFA check
+		if err != nil && err.Error() == "mfa required" {
+			fmt.Printf("Verification code: ")
+			vc, _ := gopass.GetPasswdMasked()
+			verificationCode = string(vc)
+
+			token, err = m.Login(username, password, verificationCode)
+			if err != nil {
+				h.PrintError("Authentication failed")
```

Comment:
To keep error messages as expected on the login, it may look better to keep inside the conditional only the logic for the verification code, so you only have one call to `Login` method, something like:
```
                verificationCode = ""

		// MFA check
		if err != nil && err.Error() == "mfa required" {
			fmt.Printf("Verification code: ")
			vc, _ := gopass.GetPasswdMasked()
			verificationCode = string(vc)
		}

                token, err = m.Login(username, password, verificationCode)
		if err != nil {
			h.PrintError(err.Error())
		}
```